test/planFinder.test.js

```diff
@@ -627,7 +635,7 @@ describe("E2E tests with Pipeline Engine", function() {
         }]);
     });
     it("Runs a pipeline mimicking the sensei use case - sensei doesn't support gif", async function() {
-        const pipeline = new Engine();
+        const pipeline = new Engine(params);
```

Comment:
I'd like to see some of those tests not have the `params` var set, to test error handling.

Reply:
added